packages/devtools_app/lib/src/shared/navigation.dart

```diff
@@ -17,17 +15,17 @@ import 'theme.dart';
 ///
 /// This will fail because we can't determine what the theme value for the app
 /// is.
-String routeNameWithQueryParams(BuildContext context, String routeName,
-    [Map<String, String> queryParameters]) {
+String routeNameWithQueryParams(BuildContext? context, String routeName,
+    [Map<String, String>? queryParameters]) {
```

Comment:
nit: trailing comma

Reply:
I accidentally addressed this comment in https://github.com/flutter/devtools/pull/3657